x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/product_selector/product_selector.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiPageContentBody,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { APP_SEARCH_PLUGIN, WORKPLACE_SEARCH_PLUGIN } from '../../../../../common/constants';
+
+import { SetEnterpriseSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
```

Comment:
Why do we alias this? I feel like it confuses things a bit.

Reply:
Copying Constance's pattern from other components. If we want to change IMO that is beyond the scope of this PR and we should discuss as a team and change all in a separate PR

Reply:
Hey! This pattern's on me (also one we use in multiple places, e.g., with telemetry, so def open to discussing and refactoring separately from this PR).

Most of these components `Set<Product>Chrome`, `Set<Product>Telemetry` are a very very light wrapper around what's essentially a generic function with a product key passed in. Examples:

- [Breadcrumbs](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/shared/kibana_chrome/generate_breadcrumbs.ts#L57-L78)
- [Document titles](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/shared/kibana_chrome/generate_title.ts#L27-L38)
- [Telemetry](https://github.com/elastic/kibana/blob/master/x-pack/plugins/enterprise_search/public/applications/shared/telemetry/send_telemetry.tsx#L37-L39)

I thought it would feel and read more nicely from a dev experience perspective - for example, if you're an App Search dev you know you're already working in the App Search plugin, so no need to be overly specific when calling the component - just rename it to a more generic `<SetChrome />` component, or `<SendTelemetry />`, etc. without having to repeat App Search or pass the app search ID again and again.

Definitely happy to discuss the overall pattern and if you have any thoughts on refactoring, but would very likely be in a separate PR if that sounds cool!